common/supplemental/supplementalData.xml

```diff
@@ -2604,30 +2604,84 @@ XXX Code for transations where no currency is involved
+			<languagePopulation type="ak" populationPercent="0.050" references="R1329"/>	<!--Akan-->
+			<languagePopulation type="ojs" populationPercent="0.040" references="R1329"/>	<!--Oji-Cree-->
+			<languagePopulation type="chp" populationPercent="0.034" officialStatus="official_regional" references="R1329"/>	<!--Chipewyan-->
+			<languagePopulation type="moe" populationPercent="0.032" references="R1329"/>	<!--Innu-aimun-->
+			<languagePopulation type="yi" populationPercent="0.031" references="R1329"/>	<!--Yiddish-->
+			<languagePopulation type="cr" populationPercent="0.025" officialStatus="official_regional" references="R1329"/>	<!--Cree-->
+			<languagePopulation type="mic" populationPercent="0.022" references="R1329"/>	<!--Mi'kmaq-->
+			<languagePopulation type="atj" populationPercent="0.018" references="R1329"/>	<!--Atikamekw-->
+			<languagePopulation type="bla" populationPercent="0.014" references="R1329"/>	<!--Siksika-->
+			<languagePopulation type="crk" populationPercent="0.012" references="R1329"/>	<!--Plains Cree-->
+			<languagePopulation type="den" populationPercent="0.0064" officialStatus="official_regional" references="R1329"/>	<!--Slave-->
+			<languagePopulation type="dgr" populationPercent="0.0059" officialStatus="official_regional" references="R1329"/>	<!--Dogrib-->
+			<languagePopulation type="csw" populationPercent="0.0050" references="R1329"/>	<!--Swampy Cree-->
+			<languagePopulation type="moh" populationPercent="0.0049" references="R1329"/>	<!--Mohawk-->
+			<languagePopulation type="nsk" populationPercent="0.0039" references="R1329"/>	<!--Naskapi-->
+			<languagePopulation type="dak" populationPercent="0.0034" references="R1329"/>	<!--Dakota-->
+			<languagePopulation type="clc" populationPercent="0.0024" references="R1329"/>	<!--Chilcotin-->
+			<languagePopulation type="hur" populationPercent="0.0020" references="R1329"/>	<!--Halkomelem-->
+			<languagePopulation type="war" populationPercent="0.0019" references="R1329"/>	<!--Waray-->
+			<languagePopulation type="crg" populationPercent="0.0019" references="R1329"/>	<!--Michif-->
+			<languagePopulation type="lil" populationPercent="0.0015" references="R1329"/>	<!--Lillooet-->
+			<languagePopulation type="pqm" populationPercent="0.0014" references="R1329"/>	<!--Malecite-->
+			<languagePopulation type="oka" populationPercent="0.0014" references="R1329"/>	<!--Okanagan-->
+			<languagePopulation type="kwk" populationPercent="0.0011" references="R1329"/>	<!--Kwakʼwala-->
+			<languagePopulation type="crl" populationPercent="0.0011" references="R1329"/>	<!--Northern East Cree-->
+			<languagePopulation type="gwi" populationPercent="0.0008" officialStatus="official_regional" references="R1329"/>	<!--Gwichʼin-->
+			<languagePopulation type="crm" populationPercent="0.0003" references="R1329"/>	<!--Moose Cree-->
+			<languagePopulation type="crj" populationPercent="0.0001" references="R1329"/>	<!--Southern East Cree-->
```

Comment:
I'm ok with this for now.

Longer term, I think we need to consider a somewhat higher bar for what we include. My concern is that people might expect the same level of detail for all countries.  That is if we list a language with (say) 35 speakers in CA, does that mean that if someone looks at (say) Ukraine, would they expect that every language with over 35 speakers would be listed?

Anyway, I think this needs more discussion (and not on this PR). We might put some strong qualifications into the spec and data file so that people are not mislead, or instead of listing communities of (say) < 10,000 we might include a reference to the source document.


100.0000% | 35,121,728 (for Canada, 2016)
-- | --
10.0000% | 3,512,173
1.0000% | 351,217
0.1000% | 35,122
0.0100% | 3,512
0.0010% | 351
0.0001% | 35

Reply:
The problem is that (1) clients want to programmatically get data on language usage (so pointing off somewhere else is not useful), and (2) some clients (at least ones we have) have a strong interest in support for indigenous languages. And some Canadian provinces (e.g. Northwest Territories) designate some languages with just a few hundred speakers as official in those provinces. So I would like to preserve the indigenous language data in this case. But what I can do (as only as I am fixing the data for French and English) is use a higher threshold for the non-indigenous languages and reduce those somewhat.